packages/frontend/app/services/contact-manager.ts

```diff
@@ -10,26 +9,23 @@ import ArrayProxy from '@ember/array/proxy';
 export default class ContactManager extends Service {
```

Comment:
This file has some reorganization and some changes around test stability improvements